src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>Register</title>
+ <link rel="stylesheet" href="/css/main.css">
+ <style>
+   label {
+     display: inline-block;
+     width: 100px;
+   }
+ </style>
+</head>
+<body>
+
+ <nav>
+   <a id="navTitle" href="/">CodeU Chat App</a>
+   <a href="/conversations">Conversations</a>
+   <% if(request.getSession().getAttribute("user") != null){ %>
+     <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+   <% } else{ %>
+     <a href="/login">Login</a>
+   <% } %>
+ </nav>
+
+ <div id="container">
+   <h1>Register</h1>
+
+   <form action="/register" method="POST">
+     <label for="username">Username: </label>
+     <input type="text" name="username" id="username">
+     <br/>
+     <label for="password">Password: </label>
+     <input type="password" name="password" id="password">
```

Comment:
Are we still going to add the confirm password field? Also, you should probably include the instructions on what is expected for the username and password. (8 char min with one cap etc)

Reply:
Going to approve now and you can make the changes when you (or whoever is doing it) also adds the logic for the submit button.